itests/qtest-iceberg/pom.xml

```diff
@@ -122,6 +122,12 @@
       <artifactId>jersey-servlet</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.roaringbitmap</groupId>
```

Comment:
Where is this used?

Reply:
The q test fails with ClassNotFoundException if this is not here. It's the same dependency included into the handler module: https://github.com/apache/hive/blob/master/iceberg/iceberg-handler/pom.xml#L103-L108